packages/trie/examples/merkle_patricia_trees/README.md

```diff
@@ -601,48 +596,68 @@ Easy, wasn't it?
 Merkle trees allow us to verify the integrity of their data contained without requiring us to download the full tree. To demonstrate this we will re-use the example above. As we saw, we had an extension node that pointed to a branch node:
 
 ```jsx
-console.log('Value of extension node: ', node2)
+console.log('Extension node: ', node2)
 
-const node3 = await trie.lookupNode(Buffer.from(node2._value))
-  console.log(node3)
+const node3 = await trie.lookupNode(node2._value)
+console.log(node3)
 
 // RESULT
 
-Value of extension node:  ExtensionNode {
+Extension node: ExtensionNode {
   _nibbles: [ 0, 3, 0, 3, 0 ],
-  _value: <Buffer 70 b3 d0 20 ad 85 8f d6 00 28 a4 23 e9 8f 1d 99 c5 37 cd b9 1f 27 49 16 40 06 6f ea c7 9c 2f 72>
+  _value: Uint8Array(32) [
+    112, 179, 208,  32, 173, 133, 143, 214,
+      0,  40, 164,  35, 233, 143,  29, 153,
+    197,  55, 205, 185,  31,  39,  73,  22,
+     64,   6, 111, 234, 199, 156,  47, 114
+  ],
+  _terminator: false
 }
 
 BranchNode {
   _branches: [
-    <Buffer >,
-    <Buffer >,
-    <Buffer >,
-    [ <Buffer 31>, <Buffer 74 65 73 74 56 61 6c 75 65 31> ],
-    [ <Buffer 31>, <Buffer 74 65 73 74 56 61 6c 75 65 41> ],
-    <Buffer >,
-    <Buffer >,
-    <Buffer >,
-    <Buffer >,
-    <Buffer >,
-    <Buffer >,
-    <Buffer >,
-    <Buffer >,
-    <Buffer >,
-    <Buffer >,
-    <Buffer >
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  [
+    Uint8Array(1) [ 49 ],
+    Uint8Array(10) [
+      116, 101, 115, 116,
+       86,  97, 108, 117,
+      101,  49
+    ]
+  ],
+  [
+    Uint8Array(1) [ 49 ],
+    Uint8Array(10) [
+      116, 101, 115, 116,
+       86,  97, 108, 117,
+      101,  65
+    ]
   ],
-  _value: <Buffer >
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) []
+],
+  _value: Uint8Array(0) []
 }
 
 ```
 
 Now let's suppose that I'm provided with various pieces of information, some that I trust, and some that I don't. Here's what I know from a trustworthy source:
 
 - The tree contains key-value pair "testKey": "testValue"
-- There also exists a branch node with hash: `<Buffer 70 b3 d0 20 ad 85 8f d6 00 28 a4 23 e9 8f 1d 99 c5 37 cd b9 1f 27 49 16 40 06 6f ea c7 9c 2f 72>`.
+- There also exists a branch node with hash (the value of the extension node in hex): `0x70b3d020ad858fd60028a423e98f1d99c537cdb91f27491640066feac79c2f72`.
```

Comment:
I'm not sure what "the value of the extension node in hex" is referring to, here. Could you clarify or reword it?

Reply:
I've improved the comment, hopefully that clarifies it.